build_docs.sh

```diff
@@ -14,7 +14,7 @@ esac
 WSDIR=$(mktemp -d)
 cd ${WSDIR}
 mkdir src
-wget -O ros2.repos https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos
+wget -O ros2.repos https://raw.githubusercontent.com/ros2/ros2/${RELEASE_NAME}/ros2.repos
```

Comment:
This will fetch the repos file with development branches rather than the currently released versions. Is that intended or should the `release-$RELEASE_NAME` ref be used instead?

Reply:
Good question. I guess in this context it makes sense to grab the released versions, since this will be what we ship to users. Updated in af5afc3

Reply:
err, `${RELEASE_NAME}-release` (d3ba6ff)